docs/docs/components/radio.mdx

```diff
@@ -0,0 +1,16 @@
+# Radio (spec)
+
+# Notes
+
+- Single radio component.
+- Will contain Label.
+- Will have to provide some type of prop to configure the color of the selected radio button.
+
+| Attributes  |  Values  | Optional ? |
+| :---------- | :------: | ---------: |
+| `id`        |  string  |        Yes |
+| `name`      |  string  |         No |
+| `value`     |  string  |         No |
+| `label`     |  string  |         No |
+| `isChecked` | boolean  |         No |
+| `onChange`  | function |         No |
```

Comment:
other props look great, can we expose more props to customize the styling aspect of the component? eg: changing the color.

Also do we need a radioGroup? as other libraries use. Any merits demerits of those?

Reply:
- Yeah about the color i wanted to have a discussion . Is there a mock for different color's and how much customisation should we offer in terms of color . 
- About the Radio Group there are merits and demerits as well.
     1. Since most of the cases radio will be used in a group it makes sense.Very rarely we would use a single radio button on its own.
     2. But if do have radio group we should make it robust. Like a vertical aligned radio group, or horizontal aligned radio group and all kinds of customisation.But not really sure if this is what hover is looking for now.
     3. Yes over the long radio group will make a lot of sense.